melanie-lab/src/component/modal/index.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+import './_modal.scss';
+import React, { Component } from 'react';
+
+export default class Modal extends Component {
+  render() {
+    if (!this.props.show) {
+      return null;
+    } else {
+      return (
+        <section className='modal'>
+          <main>
```

Comment:
If you think of your app as a whole as an html page, you should really only have one Main, and this modal probably wouldn't count semantically as the _main_ thing